assets/css/components/_brand.scss

```diff
@@ -1,10 +1,9 @@
 @font-face {
 	font-family : 'cloudinary';
-	src         : url('fonts/cloudinary.eot');
-	src         : url('fonts/cloudinary.eot?x#iefix') format('embedded-opentype'),
-	url('fonts/cloudinary.ttf') format('truetype'),
-	url('fonts/cloudinary.woff') format('woff'),
-	url('fonts/cloudinary.svg?x#cloudinary') format('svg');
```

Comment:
Restored this.

Reason for these small changes in the css/svg is because I had some issues in the beginning when settings up the new webpack config, so I was trying different things and forgot to restore them. Great catches!